config/application.rb

```diff
@@ -22,5 +22,9 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.exceptions_app = self.routes
+    
+    # This disables the backtrace stuffs and renders custom methods.
+    config.consider_all_requests_local = false
```

Comment:
I think we might want to leave this in their respective environment files

It will be good for developers to see the trace of any error they are getting while developing, and then we only set it to false on prod.